CIP-0067/README.md

```diff
@@ -0,0 +1,128 @@
+---
+CIP: 67
+Title: Asset Name Label Registry
+Authors: Alessandro Konrad <alessandro.konrad@live.de>, Thomas Vellekoop <thomas.vellekoop@iohk.io>
+Status: Draft
+Type: Informational
+Created: 2022-07-13
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This proposal defines a standard to classify Cardano native assets by the asset name.
+
+## Motivation
+
+As more assets are minted and different standards emerge to query data for these assets, it's getting harder for 3rd parties to determine the asset type and how to proceed with it. This standard is similar to [CIP-0010](../CIP-0010), but focuses on the asset name of an asset.
+
+
+## Specification
+
+To classify assets the `asset_name` needs to be prefixed the following `4 bytes` binary encoding:
+```
+[ 0000 | 2 bytes label_num | 1 byte checksum | 0000 ]
```

Comment:
```suggestion
[ 0000 | 16 bits label_num | 8 bits checksum | 0000 ]
```

Since the bracket is given in bits, I find it more logical to speak about bits for the inner parts as well.